addons/survey/static/src/js/survey_form.js

```diff
@@ -921,71 +938,30 @@ publicWidget.registry.SurveyFormWidget = publicWidget.Widget.extend(SurveyPreloa
     * Initialize datetimepicker in correct format and with constraints
     */
     _initDateTimePicker: function ($dateGroup) {
-        var disabledDates = [];
+        // TODO: Could maybe replaced by data-widget="date(time)-picker" ?
         var questionType = $dateGroup.find('input').data('questionType');
-        var minDateData = $dateGroup.data('mindate');
-        var maxDateData = $dateGroup.data('maxdate');
-
-        var datetimepickerFormat = questionType === 'datetime' ? time.getLangDatetimeFormat() : time.getLangDateFormat();
-
-        var minDate = minDateData
-            ? this._formatDateTime(minDateData, datetimepickerFormat)
-            : moment({ y: 1000 });
-
-        var maxDate = maxDateData
-            ? this._formatDateTime(maxDateData, datetimepickerFormat)
-            : moment().add(200, "y");
-
-        if (questionType === 'date') {
-            // Include min and max date in selectable values
-            maxDate = moment(maxDate).add(1, "d");
-            minDate = moment(minDate).subtract(1, "d");
-            disabledDates = [minDate, maxDate];
-        }
-
-        $dateGroup.datetimepicker({
-            format: datetimepickerFormat,
-            minDate: minDate,
-            maxDate: maxDate,
-            disabledDates: disabledDates,
-            useCurrent: false,
-            viewDate: moment(new Date()).hours(minDate.hours()).minutes(minDate.minutes()).seconds(minDate.seconds()).milliseconds(minDate.milliseconds()),
-            calendarWeeks: true,
-            icons: {
-                time: 'fa fa-clock-o',
-                date: 'fa fa-calendar',
-                next: 'oi oi-chevron-right',
-                previous: 'oi oi-chevron-left',
-                up: 'oi oi-chevron-up',
-                down: 'oi oi-chevron-down',
+        let minDate = $dateGroup.data('mindate');
+        let maxDate = $dateGroup.data('maxdate');
+        const [input] = $dateGroup.find("input");
+
+        const [parse, deserialize] =
+            questionType === "date"
+                ? [parseDate, deserializeDate]
+                : [parseDateTime, deserializeDateTime];
+        minDate = minDate ? deserialize(minDate) : DateTime.fromObject({ year: 1000 });
+        maxDate = maxDate ? deserialize(maxDate) : DateTime.now().plus({ year: 200 });
+        const value = parse(input.value);
+
+        this.call("datetime_picker", "create", {
+            target: input,
+            pickerProps: {
+                type: questionType,
+                // FIXME new date picker does not support limit DateTimes and will only use the date
```

Comment:
is this a real issue, do we have a use case ?

Reply:
In survey, for questions of type datetime you can select a max and min datetime, and I can only assume that this means we want to prevent people from picking datetimes outside this range.

Reply:
Yes there is dedicated datetime support in survey. Could this logic be added to the new picker at this time please?